provider/debug/debug.go

```diff
@@ -60,6 +60,39 @@ func (p *Provider) MVTLayer(ctx context.Context, layerName string, tile tegola.T
 			},
 		}
 		layer.AddFeatures(debugOutline)
+		ext1, err := tile.BufferedBoundingBox()
+		if err != nil {
+			return nil, err
+		}
+		/*
```

Comment:
purge this comment, or add a TODO to leverage the new log package.

Reply:
The new log package will still compile this code, but will not remove it from the binary. It's really only needed when I'm debugging; this algo; It's really for the clipping issues I'm dealing with in issue-179. I'll remove it after I've proven that one out.

Reply:
NM will be removing this. Was thinking this different code. Will remove.